examples/website/src/shared/utility.ts

```diff
@@ -125,7 +125,12 @@ export function transformCssStringToObject(styleCustom: string): object {
   return styleRules.reduce((previousValue: object, currentValue: string) => {
     const [key, value] = currentValue.split(':')
     if (key && value) {
-      return Object.assign(previousValue, {[key.trim()]: value.trim()})
+      return Object.assign(previousValue, {
+        [key
+          .toLowerCase()
+          .replace(/-(.)/gm, ($0, $1) => $1.toUpperCase())
+          .trim()]: value.trim()
+      })
```

Comment:
Please add a comment and describe what this function does. Nobody understands this.